src/features/newTab/default/clock/index.tsx

```diff
@@ -66,3 +66,5 @@ export class Clock extends React.PureComponent<{}, ClockState> {
     )
   }
 }
+
+export default Clock
```

Comment:
could you remove this line? we are exporting this component but not as default. see the other comments, I think we can get ride of it if all changes are applied